Portfolio.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/myStyle.css"/>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Portfolio</title>
+  </head>
+
+
+  <body>
+    <main class="content">
+      <header class="header-container">
+          <div class="Logo">
+            <img src="Logo.png" alt="log"/>
+          </div>
```

Comment:
- [ ] Please ensure that the logo is a link. You could achieve this by wrapping it inside the `<a>` tag.